source-code/plugins/i18next/src/plugin.test.ts

```diff
@@ -175,6 +175,6 @@ it("should correctly identify placeholders", async () => {
 	const resources = await config.readResources!({
 		config: config as any,
 	})
-	expect(resources[0]?.body[0]?.pattern?.elements[0]?.type).toBe("Text")
-	expect(resources[0]?.body[0]?.pattern?.elements[1]?.type).toBe("Placeholder")
+	expect(resources[0]?.body[0]?.pattern?.elements[0]?.type).toBe("Placeholder")
+	expect(resources[0]?.body[0]?.pattern?.elements[1]).toBe(undefined)
```

Comment:
@NilsJacobsen you just broke the test. 

How do you know if `Hello {{placeholder}}` will return a text element now and a placeholder? You don't :D 

Instead, either/or: 

- create a new unit test testing in isolation what you want to test
- update the test without changing the message and test that pattern.elements[2] is undefined

Reply:
In other words, you added a new unit test by removing another unit test. But, both are relevant. Thus, you should create a new unit test

Reply:
Alright got it. Give me a sec.

Reply:
Added a second test for it.